.buildkite/scripts/cloud-cleanup-oidc.sh

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env bash
```

Comment:
I copied it from .`buildkite/scripts/cloud-cleanup.sh` and added the support for OIDC.